apps/examples/testcase/le_tc/network/tc_net_pbuf.c

```diff
@@ -39,7 +39,7 @@
 * @brief		:check for the memory allocation for the pbuf by transport layer
 * @scenario		:
 * @apicovered		:pbuf_alloc()
-* @precondition 	:
+* @precondition		:
```

Comment:
I think removing a space is right. This looks it is not aligned.
(You gave a reply to unknown ID)

Reply:
yes. here it is looking not aligned but in linux machine(vim editor), it looks aligned.
now you suggest what to do?

Reply:
I think you changed a space to a tab. How about just removing a space?

Reply:
if i remove a space then at vi editor, it shows me like this
* @precondition :

Reply:
You should consider Github syntax.... `@precondtion` which you wrote is in c code. But without any other syntax, `@precondition` sends a mail to `precondition` Github ID. He already got two mail because of this.

Reply:
@deepaksrma You must take care of using `@` in Github comment.

Reply:
ok

Reply:
Don't worry about me